docs/BR.md

```diff
@@ -955,18 +1004,26 @@ In the event of reaching voicemail, the CA may leave the Random Value and the IP
 
 The Random Value SHALL remain valid for use in a confirming response for no more than 30 days from its creation. The CPS MAY specify a shorter validity period for Random Values.
 
-##### 3.2.2.5.6 ACME “http-01” method for IP Addresses
+##### 3.2.2.5.6 ACME "http-01" method for IP Addresses
+
+Confirming the Applicant's control over the IP Address by performing the procedure documented for an "http-01" challenge in draft 04 of "ACME IP Identifier Validation Extension," available at <https://tools.ietf.org/html/draft-ietf-acme-ip-04#section-4>.
```

Comment:
(Unrelated) Hmm, this (and tls-alpn-01 below) should be updated to point at the actual RFC, not draft-04, at some point...

Reply:
Also suggested [here](https://github.com/cabforum/servercert/issues/169) and [here](https://github.com/cabforum/servercert/issues/446). 

This update is planned for inclusion in the upcoming clean-up ballot (led by @barrini).